.github/workflows/tests.yml

```
@@ -17,9 +17,9 @@ jobs:
       fail-fast: false
       matrix:
         es_stack:
-          - 8.15.3
           - 8.16.0
```

this needs to be the latest minor which is 8.16.1

https://github.com/elastic/elasticsearch/releases/tag/v8.16.1

```suggestion
          - 8.16.1
```